schedule/yast/sles+sdk+proxy_SCC_via_YaST_pvm.yaml

```diff
@@ -9,26 +9,13 @@ vars:
   SYSTEM_ROLE: textmode
```

Comment:
Can delete this setting.
As you use 'installation/system_role/select_role_text_mode'

Reply:
done